sherpa/utils/__init__.py

```diff
@@ -24,16 +25,16 @@
 
 import operator
 import inspect
-from itertools import izip
+from six.moves import zip as izip
```

Comment:
Can this be moved to be with the other imports from `six.moves`? It's a minor stylistic point, so only worth doing if you feel it is worthwhile.